Model/Config/Source/ManualCaptureMethods.php

```diff
@@ -0,0 +1,61 @@
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2015 Adyen BV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+declare(strict_types=1);
+
+namespace Adyen\Payment\Model\Config\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+class ManualCaptureMethods implements OptionSourceInterface
+{
+    public function toOptionArray(): array
+    {
+        return [
+            ['value' => 'cup',               'label' => 'China Union Pay'],
+            ['value' => 'cartebancaire',     'label' => 'Carte Bancaire'],
+            ['value' => 'visa',              'label' => 'VISA'],
+            ['value' => 'visadankort',       'label' => 'VISA Dankort'],
+            ['value' => 'mc',                'label' => 'Mastercard'],
+            ['value' => 'uatp',              'label' => 'Universal Air Travel Plan'],
+            ['value' => 'amex',              'label' => 'American Express'],
+            ['value' => 'maestro',           'label' => 'Maestro'],
+            ['value' => 'maestrouk',         'label' => 'Maestro UK'],
+            ['value' => 'diners',            'label' => 'Diners Club'],
+            ['value' => 'discover',          'label' => 'Discover'],
+            ['value' => 'jcb',               'label' => 'JCB'],
+            ['value' => 'laser',             'label' => 'Laser'],
+            ['value' => 'paypal',            'label' => 'PayPal'],
+            ['value' => 'sepadirectdebit',   'label' => 'SEPA Direct Debit'],
+            ['value' => 'dankort',           'label' => 'Dankort'],
+            ['value' => 'elo',               'label' => 'Elo'],
+            ['value' => 'hipercard',         'label' => 'Hipercard'],
+            ['value' => 'mc_applepay',       'label' => 'Mastercard Apple Pay'],
+            ['value' => 'visa_applepay',     'label' => 'VISA Apple Pay'],
+            ['value' => 'amex_applepay',     'label' => 'American Express Apple Pay'],
+            ['value' => 'discover_applepay', 'label' => 'Discover Apple Pay'],
+            ['value' => 'maestro_applepay',  'label' => 'Maestro Apple Pay'],
+            ['value' => 'paywithgoogle',     'label' => 'Google Pay'],
```

Comment:
The labels are not really necessary, right? As part of some more refactoring we're working on this could be useful so we might move it elsewhere at some point. But there's no problem if we keep it here for now

Reply:
Weren't necessary for this PR, but moving these settings to a magento "config source" made sense, and consumers of that expect arrays of [value, label] object. Also means that it can easily be made configurable via system.xml in the future as a bonus